src/components/product-parameter/ProductParameter.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import LabelledInput from '../labelled-input'
+
+const ProductParameter = ({ text, options, onChange, ...other }) => {
+  const [defaultValue, setDefaultValue] = useState('Выбрать')
+  const [option, setOption] = useState({ name: 'my name', value: 'my value' })
+
+  const selectInput = (
+    <select
+      name={text}
+      onChange={() => {
+        setDefaultValue(null)
+        setOption({ value: 'Hi', name: 'Bye' })
+      }}
+    >
+      {defaultValue ? (
+        <option selected value="0">
+          {defaultValue}
+        </option>
+      ) : null}
+      {options.map((item) => {
+        const { name, value } = item
+        return (
+          <option name={name} value={value} key={name}>
```

Comment:
Also, let's pass `name` to `value` attribute instead of `value`
```suggestion
          <option name={name} value={name} key={name}>
```